Multisig/UI/Settings/OwnerKeyManagement/LedgerOwnerKey/LedgerKeyPickerViewController.swift

```diff
@@ -0,0 +1,291 @@
+//
+//  LedgerKeyPickerViewController.swift
+//  Multisig
+//
+//  Created by Andrey Scherbovich on 18.08.21.
+//  Copyright © 2021 Gnosis Ltd. All rights reserved.
+//
+
+import UIKit
+
+fileprivate let ledgerSerialQueue = DispatchQueue(label: "io.gnosis.safe.ledger.serial.queue")
+
+class LedgerKeyPickerViewController: SegmentViewController {
```

Comment:
@sche can we review this class together with you?

Reply:
sure